TaskHistoryRecord.java

```diff
@@ -0,0 +1,28 @@
+@Entity
+@Table(name = "taskhistory")
+public class TaskHistoryRecord {
+    @Id
+    private long id;
+    private String record;
+
+    public TaskHistoryRecord(String record) {
+        this.record = record;
+    }
+
+
```

Comment:
here we can change  seter/ getter methods with  @Data , this annotation will automatically add getter/setter methods